annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -0,0 +1,43 @@
+# Dekorate Autogenerated Notes
+
+To access the Helm annotations or properties you just need to have the following dependency in your
+class path:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>helm-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+Compile the project using:
+
+    mvn clean install
+
+You can find the generated Helm artifacts under: `target/classes/META-INF/dekorate/helm/<chart name>/` that should look like:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+
+How can we use it?
+
+First, make sure you have logged into a cluster with Helm enabled (in OpenShift clusters, Helm is enabled by default).
+
+Then, generate the Helm artifacts and push the image into a container registry:
+
+```shell
+mvn clean package -Ddekorate.push=true -Ddekorate.docker.registry=<container registry url> -Ddekorate.docker.group=<your group>
+```
+
+This command will push the image into the container registry to be available for the cluster when deploying.
+
+Finally, let's use Helm to deploy it into the cluster:
+
+```shell
+helm install helm-example ./target/classes/META-INF/dekorate/helm/<chart name>
```

Comment:
I propose to include also 1-2 additional examples to show how to override the values.yaml using `--set` or where you pass `my-values.yaml`

Reply:
Added